src/CMakeLists.txt

```diff
@@ -176,7 +195,10 @@ if(WIN32)
       COMMENT "Generating file to allow for easier stack traces.")
   endif()
 elseif(APPLE)
+  # TODO should we set CRASH_HANDLER and other things like LINUX does below? Why is that logic specfic to LINUX?
```

Comment:
How would you like to handle this TODO?

Reply:
If you think it can be enabled for other environments then I think t makes sense to make things uniform. I haven't looked at the code, but maybe the CRASH_HANDLER doesn't exist for other environments would be my guess?